packages/strapi-parts/src/Carousel/Carousel.stories.mdx

```diff
@@ -21,7 +22,20 @@ This is the doc of the `Carousel` component
 
 ## Carousel
 
-Description...
+Carousels are slideshow components for navigating through a set of assets. 
+
+**Best practices**
+
+- Carousels should be mostly used in forms.
```

Comment:
I'm not sure to understand why? Carousel aims to provide a visual information, it's not an input 🤔

Reply:
It's not an input, but it's a field and Fields are made to be part of forms :) (in the content manager)

Reply:
This is true for our current use case of these components, but should it be a "best practice" since this is used to display multiple/single assets?

Reply:
I mean, do we want to restrict the use of this compo in a form only?

Reply:
For now, I think so. This component has been built to only work within a form (i.e. the main layout we have almost everywhere in Strapi), and don't know how it would react elsewhere. To keep our user experience consistent and relevant, I'd personally recommend this, until we get to know what specific usages the users would require :)